common/scylla_create_devices

```diff
@@ -30,7 +30,7 @@ def create_raid(devices):
     if scylla_path.is_mount():
         print(f"{scylla_path} is already mounted. Will not run 'scylla_raid_setup'!")
         sys.exit(0)
-    run(["/opt/scylladb/scripts/scylla_raid_setup", "--raiddev", "/dev/md0", "--disks", ",".join(devices),
```

Comment:
why --raiddev is being removed ?

Reply:
This is scylladb/scylla#8219, and I sent a fix at scylladb/scylla#8220 so I dropped this change.

Reply:
o.k.